packages/fxa-payments-server/src/routes/Product/index.test.tsx

```diff
@@ -412,8 +409,35 @@ describe('routes/Product', () => {
     } = await commonCreateSubscriptionFailSetup('card_declined', message);
     fireEvent.click(getByTestId('submit'));
 
-    await findByTestId('error-card-declined');
+    await findByTestId('error-card-rejected');
     expect(queryByText(message)).toBeInTheDocument();
+
+    // hide on form change
+    fireEvent.change(getByTestId('zip'), { target: { value: '12345' } });
+    // wait for the animation to finish
+    setTimeout(() => {
+      expect(getByTestId('error-payment-submission')).not.toBeInTheDocument();
```

Comment:
Instead of a `setTimeout`, this is probably a good spot to use `waitForExpect`. For example: https://github.com/mozilla/fxa/blob/34b9bcccb5ecbc9fbc155bde2ae779962e466e3c/packages/fxa-payments-server/src/routes/Subscriptions/index.test.tsx#L330
```javascript
    await waitForExpect(() =>
      expect(cancelButton).not.toHaveAttribute('disabled')
    );
```